src/components/RepoSelectDialog.tsx

```diff
@@ -63,7 +73,13 @@ const RepoSelectDialog: React.FC<Props> = ({ open, setOpen, defaultRepo }): Reac
             <Select autoWidth labelId="notes-repo-select-label" value={repoName || defaultRepo} onChange={handleChange} disabled={isLoading(apiStatus)} label="Notes Repository">
               {repos.map(r => <MenuItem key={r.name} value={r.name}>{r.name} (<SourceBranch sx={{ verticalAlign: 'middle' }} fontSize='inherit' /> {r.default_branch || 'main'})</MenuItem>)}
             </Select>
+            <Collapse in={alertOpen}>
+                <Alert sx={{ mt:2 }} severity="warning">
+                  {`You've selected a public repository. Notes could be accessed publicly.`}
```

Comment:
Why enclose alert message text in `{` `}`. Can we remove this since its just the text without any variable/expression